ports/esp32s2/common-hal/wifi/Network.c

```diff
@@ -48,3 +48,10 @@ mp_obj_t common_hal_wifi_network_get_rssi(wifi_network_obj_t *self) {
 mp_obj_t common_hal_wifi_network_get_channel(wifi_network_obj_t *self) {
     return mp_obj_new_int(self->record.primary);
 }
+
+mp_obj_t common_hal_wifi_network_get_country(wifi_network_obj_t *self) {
+    const char* cstr = (const char*) self->record.country.cc;
+        // We know that we only want the CC thus limiting to two chars
```

Comment:
Did you get the 6 length issue sorted out?

Reply:
This is how it looks like if strlen(cstr) is used. I don't really know what to do about it.

>>> wifi.radio.ap_info.country
'DE \x01\r\x14'

Reply:
I understood what is happening here: This is the full country element, not just the .cc portion! 
D (0x44) E (0x45) " " (0x20 - interpreted as "whitespace" although this stands for "Any Environment" in Wi-Fi context)
![image](https://user-images.githubusercontent.com/5174414/103038744-829dd400-456f-11eb-8024-1145e77d538c.png)

Reply:
The space character is actually correct to refer to the "Any Environment" (0x20) opposed to just "Indoor" or "Outdoor". As per feedback by espressif and also after reviewing different documents, this appears to be correct. As strlen() reads until a \0 character, I suggest we stick to the 2 to read just the country code. I've updated the comment accordingly and no longer refer to a bug. It is mainly my misinterpretation/lack of knowledge here.